LuckyDragon/app/src/main/java/com/example/luckydragon/Activities/ViewEventActivity.java

```diff
@@ -53,6 +55,7 @@ protected void onCreate(Bundle savedInstanceState) {
         // Create view
         GlobalApp globalApp = (GlobalApp) getApplication();
         event = globalApp.getEventToView();
```

Comment:
Why can't we just do getEvent(id) and pass ViewEventActivitiy id in the intent?

Reply:
We could, but I did not write this code, Arman did. I don’t want to potentially break things so leaving it like this for now